src/main/java/org/kie/processmigration/rest/MigrationResource.java

```diff
@@ -73,6 +73,12 @@ public Response getResults(@PathParam("id") Long id) throws MigrationNotFoundExc
         return Response.ok(migrationService.getResults(id)).build();
     }
 
+    @GET
+    @Path("/{id}/results/{logId}")
+    public Response getResultLog(@PathParam("id") Long id, @PathParam("logId") Long logId) throws MigrationNotFoundException {
```

Comment:
Not thrown. Please remove
```suggestion
    public Response getResultLog(@PathParam("id") Long id, @PathParam("logId") Long logId) {
```